selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/stack/ImportAndValidateEclipseCheProjectTest.java

```diff
@@ -0,0 +1,188 @@
+  @Inject private CodenvyEditor editor;
+  @Inject private Dashboard dashboard;
+  @Inject private CreateWorkspaceHelper createWorkspaceHelper;
+  @Inject private TestWorkspaceServiceClient workspaceServiceClient;
+  @Inject private DefaultTestUser defaultTestUser;
+
+  @BeforeClass
+  public void prepare() {
+    dashboard.open();
+    createWorkspaceHelper.createWorkspaceFromStackWithoutProject(ECLIPSE_CHE, WORKSPACE_NAME);
+
+    ide.switchToIdeAndWaitWorkspaceIsReadyToUse();
+    ide.waitOpenedWorkspaceIsReadyToUse();
+  }
+
+  @AfterClass
+  public void tearDown() throws Exception {
+    workspaceServiceClient.delete(WORKSPACE_NAME, defaultTestUser.getName());
+  }
+
+  @Test
+  public void checkImportAndResolveDependenciesEclipceCheProject() {
+    // import the eclipse-che project
+    projectExplorer.waitProjectExplorer();
+    menu.runCommand(WORKSPACE, IMPORT_PROJECT);
+    importProject.waitAndTypeImporterAsGitInfo(ECLIPSE_CHE_PROJECT_URL, PROJECT_NAME);
+
+    // config the project
+    projectWizard.waitCreateProjectWizardForm();
+    projectWizard.selectTypeProject(MAVEN);
+    projectWizard.clickSaveButton();
+    loader.waitOnClosed();
+
+    // TODO it is the workaround, delete it after resolve the issue
+    // TODO https://github.com/eclipse/che/issues/10515
+    // close unexpected error information dialog and click on the 'Save' button;
+    int counterErrorDialog = 0;
+
+    while (informationDialog.waitFormIsOpened()) {
+      informationDialog.waitFormToOpen();
+      informationDialog.clickOkBtn();
+      projectWizard.clickSaveButton();
+      counterErrorDialog++;
+      loader.waitOnClosed();
+    }
+
+    System.out.println(
+        "==== The error information dialog appeared " + counterErrorDialog + " times ====");
+
+    projectWizard.waitCreateProjectWizardFormIsClosed();
+    loader.waitOnClosed();
+
+    // expand the project
+    projectExplorer.waitItem(PROJECT_NAME);
+    projectExplorer.openItemByPath(PROJECT_NAME);
+    loader.waitOnClosed();
+
+    // then open files
+    // open a java file
+    projectExplorer.quickRevealToItemWithJavaScript(
+        String.format("%s/%s", PROJECT_NAME, PATH_TO_JAVA_FILE));
+    projectExplorer.openItemByPath(String.format("%s/%s", PROJECT_NAME, PATH_TO_JAVA_FILE));
+    editor.waitActive();
+
+    // open a xml file
+    projectExplorer.quickRevealToItemWithJavaScript(
+        String.format("%s/%s", PROJECT_NAME, PATH_TO_POM_FILE));
+    projectExplorer.openItemByPath(String.format("%s/%s", PROJECT_NAME, PATH_TO_POM_FILE));
+    editor.waitActive();
+
+    // open a ts file
+    projectExplorer.quickRevealToItemWithJavaScript(
+        String.format("%s/%s", PROJECT_NAME, PATH_TO_TS_FILE));
+    projectExplorer.openItemByPath(String.format("%s/%s", PROJECT_NAME, PATH_TO_TS_FILE));
+    editor.waitActive();
+
+    // open the resolving dependencies form
+    loader.waitOnClosed();
+    mavenPluginStatusBar.waitExpectedTextInInfoPanel("Resolving project:", 120);
```

Comment:
Please create variable or constant which describes this "magic number". I mean "120"